code/Frontend/react-frontend/src/pages/AddItem.js

```diff
@@ -10,32 +10,17 @@ function EquipmentForm() {
     description: "",
     status: "",
     price: "",
-    user: "",
+    owner: "",
   });
-  function getUserId() {
-    fetch("http://127.0.0.1:5000/api/userInfo", {
-      method: "GET",
-      credentials: "include", // Include cookies in the request if using cookies for sessions
-    })
-      .then((response) => response.json())
-      .then((data) => {
-        const userId = data.userId; // Extract the user ID from the response data
-        console.log("User ID:", userId);
-      })
-      .catch((error) => console.error("Error:", error));
-  }
 
   const handleSubmit = () => {
-    // Get the user ID obtained during login
-    const userId = getUserId(); // Implement a function to retrieve the user ID
-
     // Create the equipment object with the user's ID as the owner
     const newEquipment = {
       name: formData.name,
       description: formData.description,
       status: formData.status,
       price: formData.price,
-      owner: userId, // Include the user's ID as the owner
+      owner: formData.owner, // Include the user's ID as the owner
```

Comment:
In terms of the inline comment: // Include the user's ID as the owner 
I think it should be owner's name instead of user's ID based on the table I am seeing in our SQL.

Reply:
Yeah This is the next feature I have to tinker with is just taking the user of who ever is logged in -- I might need to update that field to take in the user ID since there could be multiple people with that name but for now this works in adding in items